exercises/00-init/src/modules/users/users-actions.js

```diff
@@ -1,17 +1,14 @@
+import { createActions } from 'reduxsauce'
 
 
-const ADD_USER = 'ADD_USER'
-
-const addUser = (firstName, lastName) => ({
-    type: ADD_USER,
-    payload: {firstName, lastName}
-})
-
-export default {
-    Types: {
-      ADD_USER
+const { Types, Creators } = createActions(
+    {
+        addUser: ['firstName', 'lastName'],
     },
-    Creators: {
-        addUser
+    {   
+        prefix: "users",

```

Comment:
If you try to click the button and check redux devtools, you will see that the action that was dispatched is of type `"usersADD_USER"` which is not very nice. I would consider using prefix `"USERS_"` and also renaming the action to simply `add`, so you will end up with `"USERS_ADD"`. The downside of this is that the action creator itself is just `add` instead of `addUsers` so in some contexts it might not be clear what is actually being added. In any case, you can "rename" its usage in the connected components like this:
```
addUser: (firstName, lastName) => dispatch(UsersActions.Creators.add(firstName, lastName))
```
which is fine I guess. Another option is not to use the prefix option and just handle the proper naming yourself.

Reply:
I added the prefix because it was mentioned in the exercise. I think in this context there's not going to be a lot of other actions to confuse it with, so I'll do it as you suggest.

Reply:
Oh sorry, didn't notice it in the specification. All good then.